moped-editor/src/views/projects/projectView/ProjectFunding/ProjectFundingTable.js

```diff
@@ -89,6 +88,12 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+/*
+ * Transportation Project Financial Codes
+ */
+const SOCRATA_ENDPOINT =
+  "https://data.austintexas.gov/resource/bgrt-2m2z.json?$limit=9999";
```

Comment:
this used to be in the FundingDeptUnitAutocomplete, it is called by the custom hook `useSocrataJson`. I think its better for it be in this parent component instead of the data being requested *every time* the field goes into edit state (as it does now in prod), but doing like this means its also requested even when users only visit this table without editing

Reply:
Thanks for calling out this change! I think this is ok too since this dataset only updates once per day. 

If we run into issues in the future, we could always create a wrapper component to render in the edit cell that handles the API call and feeds them to your nice, shiny, and new reusable lookup autocomplete! ✨

Reply:
agree with what's been said. this is preferred. 

as an aside it probably wouldn't hurt to add another two 9's to that limit param to make it forever proof